### ISSUE TITLE
Wont create project - errors on startup

### ISSUE THREAD

**Issue by packytagliaferro** [NativeScript/sidekick-feedback#20]
So basically I just installed sidekick and updated Nativescript CLI per instructions to on Sidekick startup. Once I got in I have the following errors:

```
[17-08-03 14:55:36.241] (Info) Version: 0.2.0-v.2017.7.31.4
[17-08-03 14:55:36.484] (Info) Checking for updates...
[17-08-03 14:55:37.033] (Info) Dependencies versions: NodeJS: 7.2.1, npm: 4.0.5, NativeScript CLI: 3.1.3.
[17-08-03 14:55:37.265] (Info) No updates available.
[17-08-03 14:55:37.425] (Info) Devices service initialized.
[17-08-03 14:55:40.820] (Error) ab.cliExtensions.shell.extensionInstalled - {"extensionName":"nativescript-starter-kits","extensionVersion":"0.1.11","error":"Command npm failed with exit code 1"}
[17-08-03 14:55:58.155] (Error) ab.cliExtensions.shell.extensionInstalled - {"extensionName":"nativescript-cloud","extensionVersion":"0.11.6","error":"Command npm failed with exit code 1"}
[17-08-03 14:55:58.526] (Error) ab.errorHandler.shell.error - Cannot read property 'isUserLoggedIn' of undefined
[17-08-03 14:55:58.532] (Error) ab.errorHandler.shell.error - Cannot read property 'getTemplates' of undefined
```

The version can be found in the top of the log. I am running a Mac with Sierra 10.12.6

**Comment by bundyo** [comment]
Do you have a firewall or corporate firewall there? Can you try running npm from command line and try installing a package?

Can you also send the error log, located in ~/Library/Application Support/NativeScriptSidekick/Logs? 

Finally, please use the template placed in the New issue text box when logging a bug, as it contains everything needed with which we can decide faster what the problem is.

**Comment by dtopuzov** [comment]
@packytagliaferro I have no idea what is the root cause of the issue, but I would recommend update to newer `npm4` version (just run `npm i -g npm@4`).

**Comment by packytagliaferro** [comment]
@bundyo Sorry, I thought I had most of the relevant information from the template. Here are the logs:[https://gist.github.com/packytagliaferro/e126a699f126fb5186510dc36b6d5840](Logs)

As for Firewalls, I dont have any and I can download things just fine using NPM from my terminal.

**Comment by packytagliaferro** [comment]
@bundyo @dtopuzov  Problem was solved by updating NPM to version 4.6.1. 

Just ran: `sudo npm i -g npm@4`
